### DISCUSSION TITLE
Please share a list of recommended residential proxy providers

### DISCUSSION THREAD

**Discussion by SkullFace141** [engageub/InternetIncome#14] [General]
Please share a list of recommended residential proxy providers

**Comment by engageub** [comment]
Moving this to discussions since this is not an issue. There are multiple residential proxy providers which can be searched through google or other search engines. There are proxy providers which are specific to a particular region and may not be in the top list of the local searches.  There are proxy providers who only sell to the residents of particular region. 
If hope you are already using them, so please share the ones from where you purchased.
Will be keeping this discussion open for others to share their experiences from different proxy providers. 

Thank you

**Comment by engageub** [comment]
Please see the list and sign up using the links mentioned to get discounted prices. More to be added soon.
https://github.com/engageub/InternetIncome/wiki/Proxies-and-Hosting-Providers

Thank you

**Reply by engageub** [reply]
Officially supported for bandwidth sharing apps. Check out the link here. https://www.proxyfish.com/billing/aff.php?aff=449

**Reply by FastAndFunky** [reply]
Is there any chance to use public proxies? I suppose such are blocked or flagged for fraud by Honeygain etc?

**Reply by engageub** [reply]
Public proxies/DataCenter Proxies are not allowed for Honeygain as of now. Only residential proxies are allowed.
Please have a look at the table for the apps which accept datacenter proxies.
You can try that and have a look at the error messages shown by the apps related to proxies. 
https://github.com/engageub/InternetIncome?tab=readme-ov-file#judge-comparison-of-different-apps-based-on-proxy-type

Thank You

**Comment by engageub** [comment]
Test branch now supports multi VPN and multi IP.
VPN providers list has been added to the following link. 

https://github.com/engageub/InternetIncome/wiki/Proxies-and-Hosting-Providers

Thank you
